### PR TITLE
[7103] Test for iticket create with relative paths (main)

### PR DESCRIPTION
**Pull request by Nishant2022**
Connected to https://github.com/irods/irods_client_icommands/pull/395

### CODE REVIEW

**Comment by Nishant2022** [comment]
See https://github.com/irods/irods_client_icommands/pull/395#issuecomment-1619051671

File: scripts/irods/test/test_iticket.py
Comment (alanking) [inline scripts/irods/test/test_iticket.py:781]: Is this test class cleaning up the collections/data objects after it's done? Even when it fails?

Consider using the "session collection" for the active user which will always be cleaned up by the test framework. If that's not feasible due to the relative nature of the paths in the test, let's wrap the test with a try/finally block which cleans up the collections afterwards.

File: scripts/irods/test/test_iticket.py
Comment (alanking) [inline scripts/irods/test/test_iticket.py:792]: Should a different user use the tickets in this test? Should we run the tests as a rodsuser?

File: scripts/irods/test/test_iticket.py
Comment (Nishant2022) [reply scripts/irods/test/test_iticket.py:781]: I believe it is necessary to create some collections in order to demonstrate possible relative paths. I will add a try/finally block to remove the collections when done.

File: scripts/irods/test/test_iticket.py
Comment (alanking) [reply scripts/irods/test/test_iticket.py]: Okay. Admin-only, full-permissions tests have bitten us in the past which is why I bring it up.

File: scripts/irods/test/test_iticket.py
Comment (Nishant2022) [reply scripts/irods/test/test_iticket.py]: Yeah I realized as soon as I pushed the commit. I pushed a second commit to fix it.

File: scripts/irods/test/test_iticket.py
Comment (korydraughn) [reply scripts/irods/test/test_iticket.py:818]: Let's clean them up. It just decreases the chance of running into errors.

Comment (Nishant2022) [reply scripts/irods/test/test_iticket.py:818]: Added cleanup

File: scripts/irods/test/test_iticket.py
Comment (alanking) [inline scripts/irods/test/test_iticket.py:824]: Might want to change `assert_icommand` to `run_icommand` just in case one fails.

**Review by alanking — COMMENTED** [review]
I think I only had one more thing and I think this is good to go

**Review by alanking — APPROVED** [review]
Very nice. Squash em and please leave off #s, and we'll have one last look

**Review by alanking — APPROVED** [review]
Looks good to me. Let's wait for one more approval and then add #s

**Review by korydraughn — APPROVED** [review]
Let's add the pounds.

**Pull request merged**
